src/models/data.h

```diff
@@ -52,7 +53,8 @@ struct Symbol
     QString path;
     // actual file path
     QString actualPath;
-    bool isKernel = false;
+    bool isKernel;
+    bool isInline;
 
     bool operator<(const Symbol& rhs) const
     {
```

Comment:
you need to handle isInline here too - and isKernel as well, apparently that's borken?